app/src/main/kotlin/compiler/analysis/BuiltinFunctions.kt

```diff
@@ -15,6 +15,15 @@ object BuiltinFunctions {
             false
         ),
 
+        // Prints a unicode string followed by a newline to output
+        "wypisz" to Function(
```

Comment:
Może „powiedz”? Tak żeby była istotna różnica z „napisz”.

Reply:
I mogłaby też być symetryczna „wysłuchaj” do wczytywania (ale to opcjonalnie, jak chcesz).